src/main.cpp

```diff
@@ -412,9 +412,27 @@ int main(int argc, char** argv) {
 
         std::stringstream footer;
         footer << "----------------------------------------------------------------------------" << std::endl;
-        footer << "Version: " << PACKAGE_VERSION << "" << std::endl;
+        footer << "Version: " << PACKAGE_VERSION << std::endl;
+        footer << "Word size: " << RAM_DOMAIN_SIZE << " bits" << std::endl;
+        footer << "Options enabled:";
+#ifdef USE_LIBFFI
+        footer << " ffi";
+#endif
+#ifdef _OPENMP
+        footer << " openmp";
+#endif
+#ifdef USE_NCURSES
+        footer << " ncurses";
+#endif
+#ifdef USE_SQLITE
+        footer << " sqlite";
+#endif
+#ifdef USE_LIBZ
+        footer << " zlib";
+#endif
+        footer << std::endl;        
         footer << "----------------------------------------------------------------------------" << std::endl;
-        footer << "Copyright (c) 2016-22 The Souffle Developers." << std::endl;
+        footer << "Copyright (c) 2016-22 The Souffle Developers." << std::endl;        
```

Comment:
Could you fix the whitespace difference here?

Reply:
Do you mean add spaces at the end of the line ?

Reply:
Sorry !
My bad, leading spaces removed.